docker-compose.preprod.yml

```diff
@@ -120,6 +117,27 @@ services:
     restart: unless-stopped
     networks:
       - elastic
+    
+  mail:
+    build:
+      context: ./src/auth
```

Comment:
I think it's the wrong context, maybe : ./src/mail ?

Reply:
i had fix it